package.json

```diff
@@ -21,7 +21,7 @@
     "stylelint": "^14.0.1"
   },
   "peerDependencies": {
-    "stylelint": "^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0"
+    "stylelint": ">=7"
   },
   "engines": {
     "node": ">=6"
```

Comment:
Like for the engines field below, you can just specify "greater than" instead of each version specifically

Reply:
The problem with this change is that there might be breaking changes in future versions, so I would always like to make sure that we actually support the version and then update peer deps version range.

Reply:
True, but in practice it's breaking either way. 

Today it breaks until you manually update the plugin, forcing people to ask you for an update. 

With this change people *might* open an issue mentioning that it doesn't work. Given the long list of supported versions, I dare to say that this is relatively unlikely.

Reply:
> True, but in practice it's breaking either way.

The difference is that the plugin does not claim to support the newest version like it would with this change. 

A major version almost always means that there are some breaking changes, and we don't know what those breaking changes are going to be.

> Today it breaks until you manually update the plugin, forcing people to ask you for an update.

This is by design, because things that this plugin uses from stylelint might break in future versions, so there needs to be a new version of this plugin to add support for a newer stylelint version. That's how it works in all the stylelint plugins and configs that I'm involved with.